src/ActionRow/index.tsx

```diff
@@ -2,17 +2,25 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 
+export interface ActionRowProps {
+  as?: React.ElementType;
+  isStacked?: boolean;
+  children?: React.ReactNode;
+  className?: string;
+}
+
```

Comment:
We decided not to change the root file (`index.jsx`) of the component to a TS file (`with .tsx extension`). We need to write type declaration only in d.ts files.